ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/CPPOutputCustomizer.java

```diff
@@ -0,0 +1,184 @@
+/*******************************************************************************
```

Comment:
ok for me lets do it. I will merge your PR. 
Can you create issue for moving console customizers?